Makefile

```diff
@@ -29,8 +29,12 @@ pkg/dssproto/dss.pb.gw.go: pkg/dssproto/dss.proto
 	protoc -I/usr/local/include -I.   -I$(GOPATH)/src   -I$(GOPATH)/pkg/mod/github.com/grpc-ecosystem/grpc-gateway@v1.9.6/third_party/googleapis   --grpc-gateway_out=logtostderr=true,allow_delete_body=true:. pkg/dssproto/dss.proto
 
 pkg/dssproto/dss.proto: install-proto-generation
-	go run github.com/NYTimes/openapi2proto/cmd/openapi2proto -spec api.yaml -annotate -out pkg/dssproto/dss.proto -tag dss -indent 2 -package dssproto
-
+	go run github.com/NYTimes/openapi2proto/cmd/openapi2proto \
+		-spec interfaces/uastech/standards/remoteid/canonical.yaml -annotate \
```

Comment:
Might be worth doing in this PR, or we can come back to it.. but if we add "operationId" to each endpoint we can get cleaner names. You can see how I did it previously here https://github.com/interuss/dss/blob/af89392461e083477be37bf9af77de1023ac526d/api.yaml. Feel free to defer to a later PR though.

Reply:
operationId tags will need to go in the standards repo which means those changes can't be in this PR.  That sounds like a good idea, but they shouldn't go in canonical.yaml -- instead, we should kick off the optionally-augmented API by branching canonical.yaml to augmented.yaml (or something like that).

Reply:
+1, ping when done and I'll adjust this PR.

Reply:
operationId tags added to newly-created `augmented.yaml`.

Reply:
Pulled in, thanks.